RUMM-3470: Update documentation for Android SDK v2 (#19062)
DataDog/documentation: content/en/real_user_monitoring/android/web_view_tracking.md

### Update your existing SDK setup

> 1. If you want to forward RUM events coming from web page, download the [latest version][2] of RUM Android SDK and setup RUM feature following the [dedicated guide][3].

Reviewer: ```suggestion
1. If you want to forward RUM events coming from web pages, download the [latest version][2] of RUM Android SDK and setup RUM feature following the [dedicated guide][3].
```

> 2. If you want to forward Log events coming from web page, download the [latest version][4] of Logs Android SDK and setup Logs feature following the [dedicated guide][5].

Reviewer: ```suggestion
2. If you want to forward Log events coming from web pages, download the [latest version][4] of Logs Android SDK and setup Logs feature following the [dedicated guide][5].
```

3. Add the Gradle dependency by declaring the `dd-sdk-android-webview` library as a dependency in the module-level `build.gradle` file:

    ```groovy
    dependencies {
        implementation "com.datadoghq:dd-sdk-android-webview:x.x.x"
    }
    ```

> 4. Enable tracking for web view you want with the following code snippet:

Reviewer: ```suggestion
4. Enable tracking for web views with the following code snippet:
```
